src/main/java/quack/Paser.java

```diff
@@ -0,0 +1,92 @@
+package quack;
+
+import quack.command.AddTaskCommand;
+import quack.command.Command;
+import quack.command.DeleteTaskCommand;
+import quack.command.ExitCommand;
+import quack.command.FindTaskCommand;
+import quack.command.ListCommand;
+import quack.command.UpdateTaskCommand;
+
+import quack.exception.InvalidCommandException;
+
+/**
+ * This class is responsible for handling user input commands.
+ */
+public class Paser {
```

Comment:
Perhaps you meant parser instead?

Reply:
Thanks for pointing this typo out, I have already changed the class name in my recent commits.